dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/Connector.java

```diff
@@ -50,6 +52,25 @@ default String getDescription() {
    */
   default void validate(ConnectorArguments arguments) throws IllegalStateException {}
 
+  default void validateDatesRange(ConnectorArguments arguments) {
+    ZonedDateTime startDate = arguments.getStartDate();
+    ZonedDateTime endDate = arguments.getEndDate();
+
+    if (startDate != null) {
+      Preconditions.checkNotNull(
```

Comment:
This check can raise NullPointerException. Is it part of IllegalStateException (which is expected of validate method)?

Reply:
Nope, it's not. We can remove the `IllegalStateException` from signature at all.

Reply:
done